netlify.toml

```
@@ -2,6 +2,7 @@
   base = "userdocs"
   publish = "userdocs/site"
   command = "cd .. && make -f Makefile.docs build-pages"
+  ignore = "git diff --quiet HEAD^ HEAD ./"
```

Might be worth being more conservative and only ever skip builds for deploy preview?
```suggestion
[context.deploy-preview]
  ignore = "git diff --quiet HEAD^ HEAD ./"
```

---

Sure, why not. :slightly_smiling_face: